submissions/2020-08-12-OpenFF-Protein-Fragments-version2/README.md

```diff
@@ -1,32 +1,54 @@
-### Description
+## Description
 
-This is the full protein fragment dataset (version2) consisting of constrained optimizations on various protein fragments prepared by David Cerutti.
+This is the full protein peptide fragment dataset (version 2.1) consisting of constrained optimizations on various protein peptide fragments prepared by David Cerutti.
 We have 12 central residues which are capped with a combination of different terminal residues.
-Details from David 
 
-The way it's set up, we are scanning phi and psi of the central residue with a random selection of ALA, GLY, SER, or VAL to the N- or C-terminus, and the customary ACE and NME blocking groups outside of that.
-### General Information
+Details from David:
+
+> The way it's set up, we are scanning phi and psi of the central residue with a random selection of ALA, GLY, SER, or VAL to the N- or C-terminus, and the customary ACE and NME blocking groups outside of that.
+
+## General Information
+
  - Date: 12/8/2020
  - Class: OpenFF constrained optimization 
  - Purpose: Optimizations 
  - Collection: OptimizationDataset
- - Name: OpenFF Protein Fragments v2.0
- - Number of unique molecules: 16
+ - Name: OpenFF Constrained Protein Fragments v2.1
+ - Number of unique molecules: 185
+ - Submitter: Josh Horton
+ 
+## Changelog
+
+Here any information regarding dataset changes are recorded.
+
+### v2.1
+
+The dataset constraints were found to be incorrect due to a mistake in the indexing.
+The indices were assumed to be 0 based but were indexed from 1; this is corrected in version 2.1.
```

Comment:
Which indices are these referring to? The provided inputs?

Reply:
Yes that's correct the raw inputs were assumed to be zero-based which caused the confusion.